### ISSUE TITLE
DropDownList does not add "required" nor "aria-required" to the element if required is set to "true"

### ISSUE THREAD

**Issue by kdikov82** [telerik/kendo-react#1253]
Currently, the only indication in the DOM when the DropDownList has "required" set to "true" is the "k-required" class. For screen readers, the "required" or the "aria-required" should be added to the element:

Possible workaround is to manually add the "aria-required" attribute:
[https://stackblitz.com/edit/react-adbbnz-ysqxza?file=app%2Fmain.jsx](https://stackblitz.com/edit/react-adbbnz-ysqxza?file=app%2Fmain.jsx)

**Comment by kdikov82** [comment]
Already fixed (version 5.5.0)
